docs/en/docs/advanced/websockets.md

```diff
@@ -137,6 +137,15 @@ With that you can connect the WebSocket and then send and receive messages:
 
 <img src="/img/tutorial/websockets/image05.png">
 
+## Handling Multiple Clients
```

Comment:
Should the title be in all lower case (as the one below)?

Reply:
You're right @teymour-aldridge, thank you! 😄